pkg/federate/kubefed/resource_test.go

```diff
@@ -227,33 +197,47 @@ func testDistribute() {
 			})
 
 			It("should return an error and not create the resource", func() {
-				err := f.Distribute(resource, clusterNames...)
+				err = f.Distribute(resource)
 				Expect(err).To(MatchError(HavePrefix("test error")))
 			})
 		})
 
 		When("the resource is not in the Kube API", func() {
 			It("should create the resource", func() {
 				// Sanity check
-				_, err := getResourceFromAPI(f.kubeFedClient, fedResource)
+				_, err = getResourceFromAPI(f.kubeFedClient, fedResource)
 				Expect(err).To(HaveOccurred())
```

Comment:
I think we should also verify the error indicates _NotFound_, ie

`Expect(apiErrors.IsNotFound(err)).To(BeTrue())`

Reply:
You got it! :)